File: food_order_app/lib/core/constants/app_fonts.dart
class AppFonts {
  AppFonts._();

  static const String roboto = 'Roboto';
  static const String hanuman = 'Hanuman';
  
  /// Helper to get the correct font family based on locale code (e.g. 'en' or 'km')
  static String getFontForLocale(String languageCode) {
    if (languageCode == 'km') {
      return hanuman;
    }
    return roboto;
  }
}

File: food_order_web/src/core/constants/app_fonts.js
export const AppFonts = Object.freeze({
  ROBOTO: 'Roboto',
  HANUMAN: 'Hanuman',
  FAMILY: "'Roboto', 'Hanuman', system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', sans-serif",

  getFontForLocale(languageCode = 'en') {
    if (languageCode === 'km') {
      return this.HANUMAN;
    }
    return this.ROBOTO;
  },
});
